.github/workflows/bindings-client-wasm.yml

```diff
@@ -1,39 +1,41 @@
-name: Wasm bindings checks
+name: Wasm bindings checks (client)
+
 on:
   push:
     branches: [develop, production]
     paths:
-      - ".github/workflows/bindings-wasm.yml"
       - ".github/actions/**"
       - "**.rs" # Include all rust files
       - "**Cargo.toml" # Include all Cargo.toml files
       - "**Cargo.lock" # Include all Cargo.lock files
       - "!**/examples/**" # Exclude all examples
       - "!**/tests/**" # Exclude all tests
-      - "!**/bindings/**" # Exclude all bindings
       - "!cli/**" # Exclude CLI
-      - "sdk/src/client/bindings/wasm/**" # Re-include wasm client bindings
-      - "sdk/src/client/bindings/nodejs/**" # Re-include node client bindings, b/c lib copied from there
```

Comment:
Nodejs binding should stay, also below on pull_requests and should also be added for the wallet binding (there the wallet nodejs binding ofc)